### PR TITLE
Bugfix/settings update

### LINKED ISSUE #3367: Refinery forms - Missing Template in admin dialogue
Hello,
I'm getting a missing template error when saving from the notification and confirmation dialogue boxes in the admin when using a refinery form. I'm using the following:
refinerycms-core (4.0.1)
refinerycms-i18n (4.0.2)
refinerycms-images (4.0.1)
refinerycms-pages (4.0.1)
refinerycms-resources (4.0.1)
refinerycms-settings (4.0.1)
and the latest refinery-form gem

This issue showed up in a current project and I replicated in a fresh install. Output below.

Thanks,
Carlos

```
Missing template refinery/registrations/admin/settings/update, refinery/admin/update with {:locale=>[:en], :formats=>[:html], :variants=>[], :handlers=>[:raw, :erb, :html, :builder, :ruby, :coffee, :jbuilder]}. Searched in: * "/vagrant/app/views" * "/vagrant/vendor/extensions/registrations/app/views" * "/home/vagrant/.rvm/gems/ruby-2.4.2/gems/refinerycms-settings-4.0.1/app/views" * "/home/vagrant/.rvm/gems/ruby-2.4.2/gems/refinerycms-authentication-devise-2.0.0/app/views" * "/home/vagrant/.rvm/gems/ruby-2.4.2/gems/devise-4.4.3/app/views" * "/home/vagrant/.rvm/gems/ruby-2.4.2/gems/refinerycms-wymeditor-2.0.0/app/views" * "/home/vagrant/.rvm/gems/ruby-2.4.2/gems/refinerycms-acts-as-indexed-3.0.0/app/views" * "/home/vagrant/.rvm/gems/ruby-2.4.2/gems/refinerycms-resources-4.0.1/app/views" * "/home/vagrant/.rvm/gems/ruby-2.4.2/gems/seo_meta-3.0.0/app/views" * "/home/vagrant/.rvm/gems/ruby-2.4.2/gems/refinerycms-pages-4.0.1/app/views" * "/home/vagrant/.rvm/gems/ruby-2.4.2/gems/refinerycms-images-4.0.1/app/views" * "/home/vagrant/.rvm/gems/ruby-2.4.2/gems/refinerycms-core-4.0.1/app/views"

Extracted source (around line #46):
44
45
46
47
48
49
              

    def find(*args)
      find_all(*args).first || raise(MissingTemplate.new(self, *args))
    end

    def find_file(path, prefixes = [], *args)

Rails.root: /vagrant

Application Trace | Framework Trace | Full Trace
/home.rvm/gems/ruby-2.4.2/gems/actionview-5.1.6/lib/action_view/path_set.rb:46:in `find'
/home.rvm/gems/ruby-2.4.2/gems/actionview-5.1.6/lib/action_view/lookup_context.rb:116:in `find'
/home.rvm/gems/ruby-2.4.2/gems/actionview-5.1.6/lib/action_view/renderer/abstract_renderer.rb:18:in `find_template'
/home.rvm/gems/ruby-2.4.2/gems/actionview-5.1.6/lib/action_view/renderer/template_renderer.rb:38:in `determine_template'
/home.rvm/gems/ruby-2.4.2/gems/actionview-5.1.6/lib/action_view/renderer/template_renderer.rb:8:in `render'
/home.rvm/gems/ruby-2.4.2/gems/actionview-5.1.6/lib/action_view/renderer/renderer.rb:42:in `render_template'
/home.rvm/gems/ruby-2.4.2/gems/actionview-5.1.6/lib/action_view/renderer/renderer.rb:23:in `render'
/home.rvm/gems/ruby-2.4.2/gems/actionview-5.1.6/lib/action_view/rendering.rb:103:in `_render_template'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_controller/metal/streaming.rb:217:in `_render_template'
/home.rvm/gems/ruby-2.4.2/gems/actionview-5.1.6/lib/action_view/rendering.rb:83:in `render_to_body'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_controller/metal/rendering.rb:52:in `render_to_body'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_controller/metal/renderers.rb:141:in `render_to_body'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/abstract_controller/rendering.rb:24:in `render'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_controller/metal/rendering.rb:36:in `render'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_controller/metal/instrumentation.rb:44:in `block (2 levels) in render'
/home.rvm/gems/ruby-2.4.2/gems/activesupport-5.1.6/lib/active_support/core_ext/benchmark.rb:12:in `block in ms'
/home.rvm/rubies/ruby-2.4.2/lib/ruby/2.4.0/benchmark.rb:308:in `realtime'
/home.rvm/gems/ruby-2.4.2/gems/activesupport-5.1.6/lib/active_support/core_ext/benchmark.rb:12:in `ms'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_controller/metal/instrumentation.rb:44:in `block in render'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_controller/metal/instrumentation.rb:87:in `cleanup_view_runtime'
/home.rvm/gems/ruby-2.4.2/gems/activerecord-5.1.6/lib/active_record/railties/controller_runtime.rb:29:in `cleanup_view_runtime'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_controller/metal/instrumentation.rb:43:in `render'
vendor/extensions/registrations/app/controllers/refinery/registrations/admin/settings_controller.rb:16:in `update'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_controller/metal/basic_implicit_render.rb:4:in `send_action'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/abstract_controller/base.rb:186:in `process_action'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_controller/metal/rendering.rb:30:in `process_action'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/abstract_controller/callbacks.rb:20:in `block in process_action'
/home.rvm/gems/ruby-2.4.2/gems/activesupport-5.1.6/lib/active_support/callbacks.rb:131:in `run_callbacks'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/abstract_controller/callbacks.rb:19:in `process_action'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_controller/metal/rescue.rb:20:in `process_action'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_controller/metal/instrumentation.rb:32:in `block in process_action'
/home.rvm/gems/ruby-2.4.2/gems/activesupport-5.1.6/lib/active_support/notifications.rb:166:in `block in instrument'
/home.rvm/gems/ruby-2.4.2/gems/activesupport-5.1.6/lib/active_support/notifications/instrumenter.rb:21:in `instrument'
/home.rvm/gems/ruby-2.4.2/gems/activesupport-5.1.6/lib/active_support/notifications.rb:166:in `instrument'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_controller/metal/instrumentation.rb:30:in `process_action'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_controller/metal/params_wrapper.rb:252:in `process_action'
/home.rvm/gems/ruby-2.4.2/gems/activerecord-5.1.6/lib/active_record/railties/controller_runtime.rb:22:in `process_action'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/abstract_controller/base.rb:124:in `process'
/home.rvm/gems/ruby-2.4.2/gems/actionview-5.1.6/lib/action_view/rendering.rb:30:in `process'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_controller/metal.rb:189:in `dispatch'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_controller/metal.rb:253:in `dispatch'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_dispatch/routing/route_set.rb:49:in `dispatch'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_dispatch/routing/route_set.rb:31:in `serve'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_dispatch/journey/router.rb:50:in `block in serve'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_dispatch/journey/router.rb:33:in `each'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_dispatch/journey/router.rb:33:in `serve'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_dispatch/routing/route_set.rb:844:in `call'
/home.rvm/gems/ruby-2.4.2/gems/railties-5.1.6/lib/rails/engine.rb:522:in `call'
/home.rvm/gems/ruby-2.4.2/gems/railties-5.1.6/lib/rails/railtie.rb:185:in `public_send'
/home.rvm/gems/ruby-2.4.2/gems/railties-5.1.6/lib/rails/railtie.rb:185:in `method_missing'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_dispatch/routing/mapper.rb:17:in `block in <class:Constraints>'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_dispatch/routing/mapper.rb:46:in `serve'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_dispatch/journey/router.rb:50:in `block in serve'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_dispatch/journey/router.rb:33:in `each'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_dispatch/journey/router.rb:33:in `serve'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_dispatch/routing/route_set.rb:844:in `call'
/home.rvm/gems/ruby-2.4.2/gems/dragonfly-1.1.5/lib/dragonfly/middleware.rb:14:in `call'
/home.rvm/gems/ruby-2.4.2/gems/dragonfly-1.1.5/lib/dragonfly/middleware.rb:14:in `call'
/home.rvm/gems/ruby-2.4.2/gems/warden-1.2.7/lib/warden/manager.rb:36:in `block in call'
/home.rvm/gems/ruby-2.4.2/gems/warden-1.2.7/lib/warden/manager.rb:35:in `catch'
/home.rvm/gems/ruby-2.4.2/gems/warden-1.2.7/lib/warden/manager.rb:35:in `call'
/home.rvm/gems/ruby-2.4.2/gems/rack-2.0.5/lib/rack/etag.rb:25:in `call'
/home.rvm/gems/ruby-2.4.2/gems/rack-2.0.5/lib/rack/conditional_get.rb:38:in `call'
/home.rvm/gems/ruby-2.4.2/gems/rack-2.0.5/lib/rack/head.rb:12:in `call'
/home.rvm/gems/ruby-2.4.2/gems/rack-2.0.5/lib/rack/session/abstract/id.rb:232:in `context'
/home.rvm/gems/ruby-2.4.2/gems/rack-2.0.5/lib/rack/session/abstract/id.rb:226:in `call'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_dispatch/middleware/cookies.rb:613:in `call'
/home.rvm/gems/ruby-2.4.2/gems/activerecord-5.1.6/lib/active_record/migration.rb:556:in `call'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_dispatch/middleware/callbacks.rb:26:in `block in call'
/home.rvm/gems/ruby-2.4.2/gems/activesupport-5.1.6/lib/active_support/callbacks.rb:97:in `run_callbacks'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_dispatch/middleware/callbacks.rb:24:in `call'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_dispatch/middleware/executor.rb:12:in `call'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_dispatch/middleware/debug_exceptions.rb:59:in `call'
/home.rvm/gems/ruby-2.4.2/gems/web-console-3.6.2/lib/web_console/middleware.rb:135:in `call_app'
/home.rvm/gems/ruby-2.4.2/gems/web-console-3.6.2/lib/web_console/middleware.rb:22:in `block in call'
/home.rvm/gems/ruby-2.4.2/gems/web-console-3.6.2/lib/web_console/middleware.rb:20:in `catch'
/home.rvm/gems/ruby-2.4.2/gems/web-console-3.6.2/lib/web_console/middleware.rb:20:in `call'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_dispatch/middleware/show_exceptions.rb:31:in `call'
/home.rvm/gems/ruby-2.4.2/gems/railties-5.1.6/lib/rails/rack/logger.rb:36:in `call_app'
/home.rvm/gems/ruby-2.4.2/gems/railties-5.1.6/lib/rails/rack/logger.rb:24:in `block in call'
/home.rvm/gems/ruby-2.4.2/gems/activesupport-5.1.6/lib/active_support/tagged_logging.rb:69:in `block in tagged'
/home.rvm/gems/ruby-2.4.2/gems/activesupport-5.1.6/lib/active_support/tagged_logging.rb:26:in `tagged'
/home.rvm/gems/ruby-2.4.2/gems/activesupport-5.1.6/lib/active_support/tagged_logging.rb:69:in `tagged'
/home.rvm/gems/ruby-2.4.2/gems/railties-5.1.6/lib/rails/rack/logger.rb:24:in `call'
/home.rvm/gems/ruby-2.4.2/gems/sprockets-rails-3.2.1/lib/sprockets/rails/quiet_assets.rb:13:in `call'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_dispatch/middleware/remote_ip.rb:79:in `call'
/home.rvm/gems/ruby-2.4.2/gems/request_store-1.4.1/lib/request_store/middleware.rb:19:in `call'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_dispatch/middleware/request_id.rb:25:in `call'
/home.rvm/gems/ruby-2.4.2/gems/rack-2.0.5/lib/rack/method_override.rb:22:in `call'
/home.rvm/gems/ruby-2.4.2/gems/rack-2.0.5/lib/rack/runtime.rb:22:in `call'
/home.rvm/gems/ruby-2.4.2/gems/activesupport-5.1.6/lib/active_support/cache/strategy/local_cache_middleware.rb:27:in `call'
/home.rvm/gems/ruby-2.4.2/gems/dragonfly-1.1.5/lib/dragonfly/cookie_monster.rb:9:in `call'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_dispatch/middleware/executor.rb:12:in `call'
/home.rvm/gems/ruby-2.4.2/gems/actionpack-5.1.6/lib/action_dispatch/middleware/static.rb:125:in `call'
/home.rvm/gems/ruby-2.4.2/gems/rack-2.0.5/lib/rack/sendfile.rb:111:in `call'
/home.rvm/gems/ruby-2.4.2/gems/railties-5.1.6/lib/rails/engine.rb:522:in `call'
/home.rvm/gems/ruby-2.4.2/gems/puma-3.11.4/lib/puma/configuration.rb:225:in `call'
/home.rvm/gems/ruby-2.4.2/gems/puma-3.11.4/lib/puma/server.rb:632:in `handle_request'
/home.rvm/gems/ruby-2.4.2/gems/puma-3.11.4/lib/puma/server.rb:446:in `process_client'
/home.rvm/gems/ruby-2.4.2/gems/puma-3.11.4/lib/puma/server.rb:306:in `block in run'
/home.rvm/gems/ruby-2.4.2/gems/puma-3.11.4/lib/puma/thread_pool.rb:120:in `block in spawn_thread'
Request

Parameters:

{"utf8"=>"✓",
 "_method"=>"patch",
 "authenticity_token"=>"FA2NchLQSkW5xKGyJWaj/3rre/pbKhFD4KWYtgYA/HfbwohiCruNWLyk4F2ODsIZxNPpXW8os8wBQ210UFqYOA==",
 "setting"=>{"value"=>"carlos@***.com"},
 "modal"=>"true",
 "dialog"=>"true",
 "id"=>"registration_notification_recipients",
 "locale"=>"en"}
```

### PR DESCRIPTION
**Pull request by bricesanchez**
This fix https://github.com/refinery/refinerycms/issues/3367

### CODE REVIEW

**Comment by mostlyclicks** [comment]
Hello Brice,
I made the adjustments to my code and it's working like a charm. I may have found another issue but want to make sure it's not something I'm doing. I'll open another issue...

Thank you,
Carlos

File: Gemfile [hunk 1]
```diff
@@ -1,8 +1,13 @@
 source "https://rubygems.org"
 
+gemspec
+
 git "https://github.com/refinery/refinerycms", branch: "master" do
-  gem "refinerycms-core"
-  gem "refinerycms-testing"
+  gem "refinerycms"
```
Comment (parndt) [inline Gemfile:6]: why does it need all of `refinerycms`?

Comment (bricesanchez) [reply Gemfile:6]: I was unable to go to the admin without the all `refinerycms`

Comment (parndt) [reply Gemfile:6]: 🤔 you should be able to..

Comment (parndt) [reply Gemfile:6]: what I mean is that everything is designed to work with just `refinerycms-core` as every other extension attaches to that. We shouldn't need the whole package just to see the admin

Comment (bricesanchez) [reply Gemfile:6]: Yes i know but i had multiple bugs with only `refinerycms-core`. I could rollback this change but we should merge the others changes.

What do you think?

Comment (parndt) [reply Gemfile:6]: I think it's ultimately fine, I just wanted to provide the commentary.

**Pull request merged**